source/Playnite/SelectableItem.cs

```diff
@@ -614,6 +655,22 @@ public bool ContainsIds(IEnumerable<Guid> ids)
             return Items.Select(a => a.Item.Id).Contains(ids);
         }
 
+        private void SearchItems()
+        {
+            Items.FindAll(x => !(bool)x.IsVisible).ForEach(x => x.IsVisible = true);

```

Comment:
I think all this could be done as one iteration over the list and setting visibility for each item based on those two new proeprties..